odrive_interface/src/odrive_interface/odrive_motor.py

```diff
@@ -0,0 +1,114 @@
+
+
+import rospy
+
+from .odrive_connection_manager import OdriveConnectionManager
+
+
+ODRIVE_CONNECTION_MANAGER = OdriveConnectionManager(serial_numbers=['2084387E304E'])
```

Comment:
Hmm here you would have to list all serial numbers, but you need them again when defining the OdriveMotor objects. Maybe pass an OdriveConnectionManager object to create_odrive so that you can built it up in de odrive_interface_node? (or the Hardware Interface when we get to that)

Reply:
I do think that would fit nicely. The placement of the OdriveConnectionManager would be flexible if we add the variable to the create_odrive